updated gimmemotifs to version 0.10.0b2

simonvh opened the pull request:
* [X] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

bgruening commented:
@simonvh I get this error during installation. Any idea what is going on here?
```
[Oct  6 21:16:30] SERR /usr/local/bin/.gcc-post-link.sh: line 10: lsb_release: command not found
[Oct  6 21:16:30] SERR /usr/local/bin/.gcc-post-link.sh: line 49: ld: command not found
[iCouldn't locate crtXXX.o in default library search paths. You may not have it  at all. It is usually packaged in libc6-dev/glibc-devel packages. We will try  to locate crtXXX.o with system installed gcc...
[Oct  6 21:16:30] SERR *** Can't install the gcc package unless your system has crtXXX.o. ***
[Oct  6 21:16:30] SOUT There is no /usr/bin/gcc
[Oct  6 21:16:30] SERR Error: Error: post-link failed for: gcc-4.8.5-6
```

simonvh commented:
I have no clue. I'll look into it and see if I can fix this.

simonvh commented:
@bgruening can you try the new version (gimmemotifs=0.10.0b4=py27pl5.22.0_1)? However, I have just removed and updated some dependencies, so that should not really matter.

I seem to have no trouble installing gimmemotifs in a clean conda environment, so I'm not sure what the issue is. Although I did find this: https://github.com/ContinuumIO/docker-images/issues/21
Are you able to install other packages that depend on gcc just fine?
